R/dash.R

```diff
@@ -411,15 +407,14 @@ Dash <- R6::R6Class(
     # ------------------------------------------------------------------------
     # convenient fiery wrappers
     # ------------------------------------------------------------------------
-    run_server = function(host = NULL, port = NULL, block = TRUE, showcase = FALSE, ...) {
+    run_server = function(host = Sys.getenv('DASH_HOST', "127.0.0.1"), 
+                          port = Sys.getenv('DASH_PORT', 8050), 
+                          block = TRUE, 
+                          showcase = FALSE, 
+                          ...) {
       if (!is.null(host)) self$server$host <- host
       if (!is.null(port)) self$server$port <- as.numeric(port)
```

Comment:
Do we still want these `is.null` checks?

Reply:
I left them in, but the effect of passing `NULL` here would just be to use port 8080 instead of 8050 (as this is the `fiery` default), while the host would still be 127.0.0.1. Makes sense to :hocho: these.

Reply:
fixed in 5b28204ad06a54520e02a46dc9d506a3aef70257